cubeseed/urls.py

```diff
@@ -75,13 +83,18 @@
 register_cluster_routes(router)
 router.register(r"farm", FarmViewSet, basename="farm")
 
+# register_conversations_route(router)
+
+# register_room_routes(router)
```

Comment:
there is some commented code, here and before , it would be better if you cleanup before merging